src/pages/contact.jsx

```diff
@@ -84,35 +93,37 @@ const Contact = () => {
             </div>
           </div>
         </div>
+      </div>
```

Comment:
@subashprj55 
Alignment of text-box `Leave us a Message...` is not aligned with other text box in the form.